vdl_tools/scrape_enrich/scraper/scrape_websites.py

```diff
@@ -545,23 +538,37 @@ def _format_scraped_sites(results):
 
 if __name__ == '__main__':
     from vdl_tools.shared_tools.database_cache.database_utils import get_session
-    urls = [
-        'https://www.vibrantdatalabs.org',
-        'https://www.spicqyxdl.com/',
-        'https://elementalimpact.com/',
-        'https://elementalimpact.com/funding-opportunities/commercial-projects/'
-    ]
+    import json
+    import argparse
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--filename', type=str, default='')
+    parser.add_argument('--skip_existing', type=int, default=1)
```

Comment:
better to use `BooleanOptionalAction` which will go to `True` or `False` based on the presence of the argument:
```
parser.add_argument('--skip_existing', action=argparse.BooleanOptionalAction, default=False)
```